spec/granite/integrators/find_or_spec.cr

```diff
@@ -0,0 +1,26 @@
+require "../../spec_helper"
+
+describe "find_or_create_by, find_or_initialize_by" do
+  it "creates on find_or_create when not found" do
+    Parent.clear
+    Parent.find_or_create_by(name: "name")
+    Parent.all.first.name.should eq("name")
+    Parent.all.first.new_record?.should eq(false)
+  end
+
+  it "uses find on find_or_create_by when it exists" do
+    Parent.find_or_create_by(name: "name")
```

Comment:
Each test should be able to be ran by itself.  I recommend performing a `clear`, `create` and then `find_or_create_by` in this test.

Reply:
Done with before_each block.